index.html

```diff
@@ -68,8 +67,8 @@
           </a>
         </div>
       </header>
-      <div class="game-content" x-show="page === ''">
-        <div class="radio-toolbar">
+      <div class="game-content items-center" x-show="page === ''">
+        <div class="radio-toolbar items-center">
```

Comment:
if we're going to switch to tailwind, we should transition away these semantic classnames eventually. The idea is to not write CSS yourself.